src/compute/src/sink/persist_sink.rs

```diff
@@ -365,10 +386,14 @@ where
 
         loop {
             tokio::select! {
-                Some(event) = desired_input.next() => {
+                Some(event) = desired_input.next_mut() => {
                     match event {
-                        Event::Data(_cap, _data) => {
-                            // Just read away data.
+                        Event::Data(cap, data) => {
+                            // Just passthrough the data.
+                            data_output.give_container(&cap, data).await;
+                            // Explicitly drop the capability to make it more clear
+                            // that we are just passing the data along.
+                            drop(cap);
```

Comment:
Nit: Having two comments and a `drop` is maybe a bit too much :D But if we want to be explicit, should we also be so in the `!active_worker` code above?

Reply:
good point!